code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -189,7 +189,7 @@ emp_act
 /mob/living/carbon/human/proc/attacked_by(var/obj/item/I, var/mob/living/user, var/def_zone)
 	if(!I || !user)	return 0
 
-	if((istype(I, /obj/item/weapon/butch/meatcleaver) || istype(I, /obj/item/weapon/twohanded/chainsaw)) && src.stat == DEAD && user.a_intent == I_HARM)
+	if((istype(I, /obj/item/weapon/butch/meatcleaver) || istype(I, /obj/item/weapon/twohanded/chainsaw) || !istype(I, /obj/item/weapon/twohanded/chainsaw/rubber)) && src.stat == DEAD && user.a_intent == I_HARM)

```

Comment:
unwinnable battle
the colon operator lets you bypass proper typecasting, and would save space here
but it's a bad thing exactly because bypassing proper typecasting